<launch>
  <arg name="input_image" default="/kinect_head/rgb/image_rect_color"/>
  <arg name="device" default="cuda:0" /> <!-- cpu or cuda -->
  <arg name="model_type" default="masa_r50" /> <!-- cpu or cuda -->
  <arg name="fp16" default="true" /> <!-- use fp16 or not -->

  <!-- deva node -->
  <node name="masa_node"
        pkg="deep_vision_ros" type="masa_node.py"
        output="screen" >
    <remap from="~input_image" to="$(arg input_image)" />
    <rosparam subst_value="true" >
        device: $(arg device)
        fp16: $(arg fp16)
        model_type: $(arg model_type)
    </rosparam>
  </node>
</launch>
